### PR TITLE
[INLONG-10201][Audit] Renamed configuration variables for clarity

### LINKED ISSUE #10201: [Improve][Audit] Rename configuration variables
### Description

Rename configuration variables

### InLong Component

InLong Audit

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by haifxu**
Fixes #10201 

### Motivation

Renamed configuration variables for clarity

### Modifications

- Rename configuration variables
- Rename audit docker variables

### Verifying this change

- [x] This change is a trivial rework/code cleanup without any test coverage.

### CODE REVIEW

**Review by doleyzi — DISMISSED** [review]
LGTM

File: inlong-audit/conf/application.properties [hunk 1]
```diff
@@ -16,15 +16,17 @@
...
 audit.config.proxy.type=pulsar
 
 # Supports common JDBC protocol
 audit.config.store.mode=jdbc
 
 # manger config
 manager.hosts=127.0.0.1:8083
-proxy.cluster.tag=default_cluster
+
+# Get Kafka or Pulsar address from the cluster tag.
```
Comment (aloyszhang) [inline inlong-audit/conf/application.properties:28]: Plz, describe this configuration directly.

Comment (haifxu) [reply inlong-audit/conf/application.properties]: Done

**Review by aloyszhang — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**